src/og/ellipsoid/Ellipsoid.js

```diff
@@ -15,6 +15,10 @@ import { LonLat } from '../LonLat.js';
  * @param {number} polarSize - Polar ellipsoid size.
  */
 class Ellipsoid {
+    /**
+     * @param {number} equatorialSize - Equatorial ellipsoid size.

```

Comment:
May be we have to choose a place where we are going to make description obout constructor options. I prefer any place, but not two places.